knative-operator/hack/009-eventing-kafka-dispatcher-monitoring-rbac.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml b/knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml
+index 93e345a3..1ddb1593 100644
+--- a/knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml
++++ b/knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml
+@@ -129,6 +129,7 @@ rules:
+       - create
+       - patch
+       - update
++      - delete
+   - apiGroups:
+       - "" # Core API Group.
+     resources:
+@@ -172,6 +173,16 @@ rules:
+     resources:
+       - rolebindings
+     verbs: *everything
++  - apiGroups:
```

Comment:
Check the eventing-kafka PR why this is needed. Kafka controller which creates the dispatcher needs to setup monitoring thus it needs some more rights.